EssentialsDiscordLink/src/main/resources/config.yml

```diff
@@ -50,3 +50,9 @@ role-sync:
   roles:
     22222222222222222: vip
     33333333333333333: booster
+
+# In what mode names should be synchronised. Available modes:
+# none - Name synchronisation disabled
+# mc-to-discord - Minecraft names will be set on Discord
+# discord-to-mc - Discord names will be set on Minecraft
+name-sync: none
```

Comment:
I would probably mention the fact that the server owner will not be able to be synchronized, as bots cannot modify their nickname.